images/images.go

```diff
@@ -0,0 +1,190 @@
+package image
+
+import (
+	"bytes"
+	"github.com/kolesa-team/go-webp/encoder"
+	"github.com/kolesa-team/go-webp/webp"
+	"github.com/nfnt/resize"
+	"image/jpeg"
+	"image/png"
+	"io"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+)
+
+var (
+	Webp = "webp"
+	Jpeg = "jpeg"
+	Png  = "png"
+)
+
+type OptFunc func(image *Image) error
```

Comment:
Better to move this one to the `options` package for better understanding what this type does